octobot_evaluators/evaluator/abstract_evaluator.py

```diff
@@ -48,6 +48,9 @@ def __init__(self):
         # Evaluation related exchange name
         self.exchange_name = None
 
+        # Evaluation related exchange id
+        self.exchange_id = None
```

Comment:
:(

Reply:
I don't like to have exchange manager id outside of Trading modules as matrix id outside of Evaluation modules.

Reply:
We need a way to know to which channel an evaluator should subscribe, for trading channels we need an exchange ID. I don't have any other solution yet, do you ?
![image](https://user-images.githubusercontent.com/9078616/72665179-eb96d680-3a05-11ea-9d6c-97c21dd626ba.png)

Reply:
There is the same problem for matrix_id in Trading module. We need to find the same fix for this two issues.